requirements/sync.in

```diff
@@ -0,0 +1,2 @@
+# requirements for sync_translations script
+PyYAML
```

Comment:
Is there a reason not to add `PyYAML` into `translations.in`?

Reply:
no, just because we had a `sync.in` file